recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "ocp" %}
+{% set version = "7.5.2beta" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # The release .tar.gz does not contain the requisite `pywrap` submodule, use 
+  # the tagged git version for now
+
+  # url: "https://github.com/CadQuery/OCP/archive/refs/tags/7.5.2-beta.tar.gz"
+  # sha256: 616303149564dae4628b7f536ccfdd245cf397eaf8b3c1b2f724d517c99ac171
+
+  git_url: https://github.com/CadQuery/OCP.git
+  git_rev: 7.5.2-beta
+
+build:
+  number: 0
+
+test:
+  imports:
+    - OCP
+
+outputs:
+  - name: ocp-devel
+    script: generate-bindings.sh
+    requirements:
+      host:
```

Comment:
hm... i'm not certain, but have the feeling some of these should go in a `build`. But